docs/cicd.md

```diff
@@ -167,11 +168,15 @@ More information on the version naming schema used can be found [here](docs/conv
 
 ## Skipping specific jobs
 
-You can control which jobs should be skipped through environment variables. Set the variable `SKIP_JOB_deliver_ios` to `"all"` to skip all jobs with the name `deliver_ios`. Set the variable to `"malte"` in order to skip jobs which use the build config `malte` and have the name `deliver_ios`. You can also set it to `"malte|integreat"` in order to match multiple build configs.
+You can control which jobs should be skipped through environment variables. 
+Set the variable `SKIP_JOB_deliver_aschaffenburg_ios` to `"aschaffenburg"` to skip the job with the name `deliver_aschaffenburg_ios`.
+You can also set it to `"malte|aschaffenburg"` in order to match multiple build configs or to `"all"` to match all build configs.
```

Comment:
Not sure if i get this right, so if i want to skip ios delivery for aschaffaneburg app, i set
SKIP_JOB_deliver_aschaffenburg_ios = aschaffenburg?

Reply:
yes

Reply:
Sounds strange but thats how it has to be right now since we decided to include the build config name in the job name :/